src/Components/Admins/index.js

```diff
@@ -1,11 +1,90 @@
+import { useEffect, useState } from 'react';
 import styles from './admins.module.css';
+import Modal from './modal/modal';
+
+const Admins = () => {
+  const [admins, setAdmins] = useState([]);
+  const [modal, setModal] = useState(false);
+
+  console.log('admins', admins);
+
+  const getAdmins = () => {
+    fetch(`${process.env.REACT_APP_API}/admins`)
+      .then((response) => {
+        if (response.status !== 200 && response.status !== 201) {
+          return response.json().then(({ message }) => {
+            throw new Error(message);
+          });
+        }
+        return response.json();
+      })
+      .then((response) => {
+        setAdmins(response.data);
+      })
+      .catch((error) => {
+        console.log('err', error);
+      });
+  };
+
+  useEffect(() => {
+    getAdmins();
+  }, []);
+
+  const updateAdmin = (id) => {
+    window.location.href = `/admins/form?_id=${id}`;
+  };
+
+  const deleteAdmin = (id) => {
```

Comment:
First, you should show the modal to request a user confirmation, if the user confirms the deletion, then make the request to the server.